/**:
  ros__parameters:

    target_frame: "livox_frame"
    sub_topic: "lio_sam/mapping/cloud_registered_raw"
    scan_topic: "scan"
    odom_topic: "lio_sam/mapping/odometry"
    transform_tolerance: 0.01
    queue_size: 6
    min_height: -0.1
    max_height: 0.1
    angle_min: -1.57079632679489661923
    angle_max: 1.57079632679489661923
    angle_increment: 0.01745329252
    scan_time: 0.1
    range_min: 0.0
    range_max: 20.0
    inf_epsilon: 1.0
    use_inf: true
    hit: 0.55
    miss: 0.49
    resolution: 0.05

      
